src/cocoa/toga_cocoa/dialogs.py

```diff
@@ -99,3 +99,37 @@ def save_file(window, title, suggested_filename, file_types=None):
     if result == NSFileHandlingPanelOKButton:
```

Comment:
At column 18: (F405) 'NSFileHandlingPanelOKButton' may be undefined, or defined from star imports: .libs

Reply:
After talking to @danyeaw, not applicable - follows the existing convention of the * import.